### DISCUSSION TITLE
Error validating : Ambiguous relation detected

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6009] [Q&A] [answered]
i have two models :

```prisma
model Currency {
    id        String   @id @default(uuid())
    name    String   @unique
}

model Card {
    id        String   @id @default(uuid())
    name          String
    localCurrency   Currency     @relation(fields: [localCurrencyId], references: [id])
    localCurrencyId String
    InternationalCurrency   Currency     @relation(fields: [InternationalCurrencyId], references: [id])
    InternationalCurrencyId String
}
```

how to solve ambiguous relation ??
thanks

**Comment by ryands17** [comment, marked as answer]
You need to add the name to relations as mentioned [here](https://www.prisma.io/docs/concepts/components/introspection#disambiguating-relations):

```prisma
model Currency {
  id                 String @id @default(uuid())
  name               String @unique
  internationalCards Card[] @relation("international")
  localCards         Card[] @relation("local")
}

model Card {
  id                      String   @id @default(uuid())
  name                    String
  localCurrency           Currency @relation("local", fields: [localCurrencyId], references: [id])
  localCurrencyId         String
  InternationalCurrency   Currency @relation("international", fields: [InternationalCurrencyId], references: [id])
  InternationalCurrencyId String
}
```

**Reply by BishoyNarouz** [reply]
Thanks so much
it works for me

**Reply by claranceliberi** [reply]
Also works for me thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
